File: PYTHON/calculadora.py
print("esto es una calculadora booleana donde decides los pasos respondiendo(verdadero o falso)")
print("ingrese el primer numero para realizar la operacion")
num1=int(input())
print("ingrese el primer segundo para realizar la operacion")
num2=int(input())
resultadoSuma=num1+num2
print("desea hacer una suma?, responda (si o no)")
respuestaSuma=(input()).lower().strip()
hacerSuma=respuestaSuma in["si","sí","s"]
if hacerSuma:
    print("el resultado de la suma es :", resultadoSuma)
else:
    print("desea hacer una resta?")
    resultadoResta=num1-num2
    respuestaResta=input().lower().strip()
    hacerResta=respuestaResta in["sí","si","s"]
    if hacerResta:
        print("el resultado de la resta es: ", resultadoResta)
    else:
        print("desea hacer una multipliacion?")
        respuestaMultiplicacion=input().lower().strip()
        hacerMultiplicacion=respuestaMultiplicacion in["sí","si","s"]
        resultadoMultiplicacion=num1*num2
        if hacerMultiplicacion:
            print("el resultado de la multiplicacion es:", resultadoMultiplicacion)
        else:
            print("le toco hacer la division mijo porque no quizo elegir ninguna de las anteriores")
            resultadoDivision=num1/num2
            print("el resultado de la division es", resultadoDivision)

File: ejercicios-frontend/JavaScript/calculadora/JS/calculadora.js
import {suma,resta,multiplicacion,division} from "./funciones.js";
let operacion = Number(prompt(`ingresa el numero correspondiente a la operacion que deseas hacer:
    [1] = suma
    [2] = resta
    [3] = multiplicacion
    [4] = division`))
while (operacion != 1 && operacion != 2 && operacion != 3 && operacion != 4 ){
    alert('numero ingresado no valido, ingresa el numero correspondiente a la operacion nuevamente')
    operacion = Number(prompt(`ingresa el numero correspondiente a la operacion que deseas hacer:
    [1] = suma
    [2] = resta
    [3] = multiplicacion
    [4] = division`))
}
if (operacion == 1){
    let num1 = Number(prompt('ingrese el primer numero para realizar la suma'))
    let num2 = Number(prompt('ingrese el segundo numero para realizar la suma'))
    let resultadoSuma = suma(num1,num2)
    alert(`el resultado de la suma es: ${resultadoSuma}`)
}
else if (operacion == 2){
    let num1 = Number(prompt('ingrese el primer numero para realizar la resta'))
    let num2 = Number(prompt('ingrese el segundo numero para realizar la resta'))
    let resultadoResta = resta(num1,num2)
    alert(`el resultado de la resta es: ${resultadoResta}`)
}
else if (operacion == 3){
    let num1 = Number(prompt('ingrese el primer numero para realizar la multiplicacion'))
    let num2 = Number(prompt('ingrese el segundo numero para realizar la multiplicacion'))
    let resultadoMultiplicacion = multiplicacion(num1,num2)
    alert(`el resultado de la multiplicacion es: ${resultadoMultiplicacion}`)
}
else{
    let num1 = Number(prompt('ingrese el primer numero para realizar la division'))
    while (num1 == 0){
        alert('no se puede dividir por 0')
        num1 = Number(prompt('ingrese el primer numero nuevamente'))
    }
    let num2 = Number(prompt('ingrese el segundo numero para realizar la division'))
    while (num2 == 0){
        alert('no se puede dividir por 0')
        num2 = Number(prompt('ingrese el segundo numero nuevamente'))
    }
    let resultadoDivision = division(num1,num2)
    alert(`el resultado de la division es: ${resultadoDivision}`)
}